docs/configuration.rst

```diff
@@ -0,0 +1,48 @@
+Configuration
+=============
+
+You can set the following options in your ``conf.py`` file.
+
+.. confval:: sharedconf_docsets
+
+   A dictionary defining the projects (docsets) that will share the configuration.
+   You must define at least one docset.
```

Comment:
I've described this as "docset" as well, but perhaps "projects' is the nomenclature that most folks understand. Probably worth reflecting that in the configuration naming too if we want to clarify